.pre-commit-config.yaml

```diff
@@ -1,4 +1,10 @@
 repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
Let's move Ruff after Black: Black might fix some things that Flake8 would complain about.

Reply:
https://beta.ruff.rs/docs/usage
> Note that Ruff's pre-commit hook should run before Black, isort, and other formatting tools.